relnotes/setup-pat.bug.md

```diff
@@ -0,0 +1,16 @@
+### The `setup` comman will not attempt to login using user/password
+
+GitHub stopped supporting login using user/password with the API, so trying to
+do so in the `setup` command is doomed to fail. Because of this, support for
+generating a Personal Access Token (PAT) for the user is now removed. Instead
+the user has to create their own PAT manually and passing it to the `setup`
+command instead.
+
+In particular the `setup` command removed the `--password` option and added
+a `--oauthtoken` option instead, and when not specified, it will ask the user
```

Comment:
I would write `an --oauthtoken` myself, but I am aware that I might be in the minority of people who don't pronounce symbols (a dash dash oauthtoken).

Reply:
```suggestion
the `--oauthtoken` option instead, and when not specified, it will ask the user
```
could help solving the ambiguity. Is it still clear/correct?